test/tests/functional/pbs_pbsnodes.py

```diff
@@ -89,3 +90,25 @@ def test_pbsnodes_aS(self):
             self.assertIn(
                 hdr, out2['out'][0],
                 "header %s not found in output" % hdr)
+
+    def test_pbsnodes_av(self):
+        """
+        This verifies the values of last_used_time in 'pbsnodes -av'
+        result before and after server shutdown, once a job submitted.
+        """
+        j = Job(TEST_USER)
+        j.set_sleep_time(1)
+        jid = self.server.submit(j)
+        self.server.accounting_match("E;%s;.*" % jid, regexp=True)
```

Comment:
No need to do a .* at the end.  accounting_match() will do a substring match.  Once you remove that, there is no need for regexp=True then.